config/keys/server.key.org

```diff
@@ -0,0 +1,18 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
As bad as this might look, these keys are self-signed and generated purely for purposes of shared testing